src/ui/public/angular-bootstrap/datepicker/day.html

```diff
@@ -1,20 +1,82 @@
-      <th><button type="button" class="btn btn-default btn-sm pull-left" ng-click="move(-1)" tabindex="-1"><i class="glyphicon glyphicon-chevron-left"></i></button></th>
-      <th colspan="{{5 + showWeeks}}"><button id="{{uniqueId}}-title" role="heading" aria-live="assertive" aria-atomic="true" type="button" class="btn btn-default btn-sm" ng-click="toggleMode()" tabindex="-1" style="width:100%;"><strong>{{title}}</strong></button></th>
-      <th><button type="button" class="btn btn-default btn-sm pull-right" ng-click="move(1)" tabindex="-1"><i class="glyphicon glyphicon-chevron-right"></i></button></th>
+      <th
+        class="kuiDatePickerNavigationCell"
+        colspan="{{7 + showWeeks}}"
+      >
+        <div class="kuiDatePickerNavigation">
+          <button
+            type="button"
+            class="kuiDatePickerNavigationButton"
+            ng-click="move(-1)"
+            tabindex="-1"
+          >
+            <span class="kuiIcon fa-chevron-left"></span>
+          </button>
+
+          <button
+            id="{{uniqueId}}-title"
+            role="heading"
+            aria-live="assertive"
+            aria-atomic="true"
+            type="button"
+            class="kuiDatePickerNavigationButton"
+            ng-click="toggleMode()"
+            tabindex="-1"
+            style="width:100%;"
+          >
+            <strong>{{title}}</strong>
+          </button>
+
+          <button
+            type="button"
+            class="kuiDatePickerNavigationButton"
+            ng-click="move(1)"
+            tabindex="-1"
+          >
+            <span class="kuiIcon fa-chevron-right"></span>
+          </button>
+        </div>
+      </th>
     </tr>
     <tr>
       <th ng-show="showWeeks" class="text-center"></th>
-      <th ng-repeat="label in labels track by $index" class="text-center"><small aria-label="{{label.full}}">{{label.abbr}}</small></th>
+      <th ng-repeat="label in labels track by $index" class="text-center">
+        <small aria-label="{{label.full}}">{{label.abbr}}</small>
+      </th>
     </tr>
   </thead>
   <tbody>
     <tr ng-repeat="row in rows track by $index">
-      <td ng-show="showWeeks" class="text-center h6"><em>{{ weekNumbers[$index] }}</em></td>
-      <td ng-repeat="dt in row track by dt.date" class="text-center" role="gridcell" id="{{dt.uid}}" aria-disabled="{{!!dt.disabled}}">
-        <button type="button" style="width:100%;" class="btn btn-default btn-sm" ng-class="{'btn-info': dt.selected, active: isActive(dt)}" ng-click="select(dt.date)" ng-disabled="dt.disabled" tabindex="-1"><span ng-class="{'text-muted': dt.secondary, 'text-info': dt.current}">{{dt.label}}</span></button>
+      <td ng-show="showWeeks" class="kuiDatePickerHeaderCell">
+        <em>{{ weekNumbers[$index] }}</em>
+      </td>
+
+      <td
+        ng-repeat="dt in row track by dt.date"
+        class="kuiDatePickerRowCell"
+        ng-class="{'kuiDatePickerRowCell-isPrecluded': dt.disabled && !dt.secondary}"
```

Comment:
Ah, that makes sense why you'd want to use a different name for that situation. Honestly, I didn't pick up the intent of the word and resorted to using the dictionary to help me out, and still came up empty handed. How do you feel about "isConstrained" or "isBlocked"? Don't let my poor vocabulary purely impact your naming decision though :)